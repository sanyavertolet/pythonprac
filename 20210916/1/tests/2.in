2 + 2, 'is' + 'four', -1.3
