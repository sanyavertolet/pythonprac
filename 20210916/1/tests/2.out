<class 'int'> <class 'str'> <class 'float'>
